swagger.yaml

```
@@ -7,6 +7,69 @@ servers:
   - url: http://localhost:8000/ # for local test
 
 paths:
+  /admin/login:
+    post:
+      summary: Login and get API Key
+      tags:
+      - admin auth
+      requestBody:
+        description: credentials
+        content:
+          application/json:
+            schema:
+              type: "object"
+              required:
+              - id
+              - password
+              properties:
+                id:
+                  type: string
+                password:
+                  type: string
+      responses:
+        '200':
+          description: "Login Successful"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                properties:
+                  token:
+                    type: "string"
+        '401' :
```

ここってスペース要らなかったりしませんか

```suggestion
        '401':
```